### PR TITLE
add markers/tags for migration tests

### LINKED ISSUE #96: add possibility to run/skip only migrations tests
Calling tests with via pytest with `--migrations` or via Django's standard test runner (which apply migrations by default when using `TransactionTestCase`) can slow down tests a bit and when running tests in CI/CD pipeline we want to get feedback as fast as possible and often it's good idea to run standard and fast unit tests in parallel with migrations tests etc.

Currently it's really hard to run/skip *only* migrations tests and the easy solution is to add:

+ `marker` to every node that requested `migrator_factory` fixture
+ `tag` to every subclass of `MigrationTestCase` - https://docs.djangoproject.com/en/3.0/topics/testing/tools/#tagging-tests

### PR DESCRIPTION
**Pull request by skarzi**
Closes #96

### CODE REVIEW

File: tests/test_contrib/test_pytest_plugin/test_plugin.py [hunk 1]
```diff
@@ -19,3 +22,44 @@ def test_call_pytest_setup_plan():
...
+        ['pytest', '--markers'],
+        stderr=subprocess.STDOUT,
+        universal_newlines=True,
+        encoding='utf8',
+    )
+
+    assert MIGRATION_TEST_MARKER in output_text
+
+
+def test_pytest_markers():
```
Comment (skarzi) [inline tests/test_contrib/test_pytest_plugin/test_plugin.py:39]: @sobolevn what do you think about using [`pytester` plugin](https://docs.pytest.org/en/latest/writing_plugins.html#testing-plugins) to add more detailed tests of our pytest plugin?

File: README.md [hunk 2]
```diff
@@ -221,6 +221,34 @@ class TestDirectMigration(MigratorTestCase):
...
+with slower migrations tests.
+
+#### pytest
+
+`django_test_migrations` adds `migration_test` marker to each test using
+`migrator_factory` or `migrator` fixture.
+To run only migrations test, use `-m` option:
+
+```bash
+pytest -m migration_test
```
Comment (sobolevn) [inline README.md:238]: Let's also give an example here how to ignore migration tests:

```bash
pytest -m migration_test  # runs only migraion tests
pytest -m "not migration_test"  # runs all except migration tests
```

File: README.md [hunk 3]
```diff
@@ -221,6 +221,34 @@ class TestDirectMigration(MigratorTestCase):
...
+
+#### unittest
+
+`django_test_migrations` adds `migration_test`
+[tag](https://docs.djangoproject.com/en/3.0/topics/testing/tools/#tagging-tests)
+to every `MigratorTestCase` subclass.
+To run only migrations tests, use `--tag` option:
+
+```bash
+python mange.py test --tag=migration_test
```
Comment (sobolevn) [inline README.md:249]: We have something similar to `-m "not ..."` in `unittest`?

File: tests/test_contrib/test_pytest_plugin/test_plugin.py (hunk 1 above)
Comment (sobolevn) [reply tests/test_contrib/test_pytest_plugin/test_plugin.py:39]: Yes, sure! Let's open a new issue for it! 👍

**Review by sobolevn — CHANGES_REQUESTED** [review]
Awesome work! 

I suggest to fix docs a bit and let's merge it! 👍

File: README.md (hunk 3 above)
Comment (skarzi) [reply README.md:249]: Yes, there is a CLI option [`--exclude-tags`](https://docs.djangoproject.com/en/3.1/ref/django-admin/#cmdoption-test-exclude-tag), I will add usage example to the docs

File: tests/test_contrib/test_pytest_plugin/test_plugin.py (hunk 1 above)
Comment (skarzi) [reply tests/test_contrib/test_pytest_plugin/test_plugin.py:39]: I have added issue #140

**Review by sobolevn — APPROVED** [review]

**Comment by sobolevn** [comment]
@skarzi do we have any blockers for a new version to be released?

**Comment by skarzi** [comment]
Probably 1 blocker which is MySQL support - https://github.com/wemake-services/django-test-migrations/issues/122#issuecomment-701208857

I will take care of it whenever I find some time

**Pull request merged**